src/index.js

```diff
@@ -92,7 +92,7 @@ function parseRule(rule) {
     };
 }
 
-// {key, value, validation}
+// Returns array of failed rules or false if all rules passed.
```

Comment:
```suggestion
/**
 * Returns array of failed rules or false if all rules passed.
 *
 * @param fieldData In the format { key, value, validation, ... }
 * @param formData See `validateForm`
 */
```

I liked your description but also wanted to keep the data format in the comment. May as well go for the real deal!

Reply:
Yes, it makes sense to have both there: the description and data format.